cdap-docs/reference-manual/source/http-restful-api/http-restful-api-v2/index.rst

```diff
@@ -33,7 +33,7 @@ The Cask Data Application Platform (CDAP) has an HTTP interface for a multitude
 
 - :doc:`Introduction: <introduction>` conventions, status codes, and working with CDAP Security
 - :doc:`Lifecycle: <lifecycle>` deploying and managing Applications and managing the lifecycle of Flows,
-  Procedures, MapReduce Programs, Workflows, and Custom Services
+  Procedures, Workers, MapReduce Programs, Workflows, and Custom Services
```

Comment:
Did this actually get added to the v2 APIs? If not, this should be reverted. Perhaps this should be in the v3 version only?